stablehlo/dialect/VhloTypes.td

```diff
@@ -90,7 +90,25 @@ def VHLO_Float64V1 : VHLO_TypeDef<"Float64V1","f64">;
 
 def VHLO_Float8E4M3FNV1 : VHLO_TypeDef<"Float8E4M3FNV1", "f8E4M3FN", "0.4.0">;
 
-def VHLO_Float8E5M2 : VHLO_TypeDef<"Float8E5M2V1", "f8E5M2", "0.4.0">;
+def VHLO_Float8E5M2V1 : VHLO_TypeDef<"Float8E5M2V1", "f8E5M2", "0.4.0">;
+
+def VHLO_FunctionTypeV1 : VHLO_TypeDef<"FunctionV1", "func"> {
```

Comment:
Other types in this file are called `VHLO_SomethingV1` rather than `VHLO_SomethingTypeV1`. Let's update this name accordingly?